charts/vineyard-operator/templates/_helpers.tpl

```diff
@@ -50,3 +50,13 @@ app.kubernetes.io/name: {{ include "vineyard-operator.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end }}
 
```

Comment:
Why changing the file permission? Looks strange to have `x` bit for a tpl file.

Reply:
These file's permission is autogenerated by the `helmify` first and I update the permission manually. Besides, the helmify doesn't overwrite the previous files and it will not happen again.